packages/repository/src/mixins/repository.mixin.ts

```diff
@@ -3,17 +3,17 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
+// FIXME(rfeng): Workaround for https://github.com/microsoft/rushstack/pull/1867
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Please remove these two lines, they should stay at L28 & L29 only (see below in this file).